assembl/static2/js/app/components/home/header/statistic.jsx

```diff
@@ -37,7 +37,7 @@ class Statistic extends React.Component {
     });
   };
   render() {
-    const { rootIdea, numParticipants, totalSentiments } = this.props.data;
+    const { rootIdea, numParticipants, totalSentiments, visitsAnalytics } = this.props.data;
```

Comment:
it would have been nice to add flow typing and a test for this component

Reply:
flow types have now been added